tests/lmeval/configs/vl_w4a16_actorder_weight.yaml

```diff
@@ -1,19 +1,20 @@
 cadence: "weekly"
-model: Qwen/Qwen2-VL-2B-Instruct
-model_class: TraceableQwen2VLForConditionalGeneration
+model: Qwen/Qwen2.5-VL-7B-Instruct
+model_class: TraceableQwen2_5_VLForConditionalGeneration
+scheme: W4A16_actorder_group
```

Comment:
Seems like this scheme still says `actorder_group` should be updated to `actorder_weight`?

Reply:
Must've been a typo left over from the previous PR. Updated, thanks for catching that